src/DatePicker/DatePicker.test.js

```diff
@@ -25,70 +25,6 @@ describe('<DatePicker />', () => {
         mount(compactRangeDatepicker);
     });
 
-    test('open/close by calendar icon button', () => {
```

Comment:
These essentially test opening/closing the calendar which should now be covered by using the Popover component itself.

Reply:
🔥